gulpfile.js

```diff
@@ -1,31 +1,36 @@
 /* eslint-env node */
 
-'use strict';
-
-const beeper = require('beeper');
-const chalk = require('chalk');
-const gulp = require('gulp');
-const log = require('fancy-log');
-const sassdoc = require('sassdoc');
-const sasslint = require("gulp-stylelint");
+import beeper from 'beeper';
+import chalk from 'chalk';
+import gulp from 'gulp';
+import log from 'fancy-log';
+import sassdoc from 'sassdoc';
+import sasslint from 'gulp-stylelint';
 
 const paths = {
   TEST_DIR: 'test/',
   SASS_DIR: 'sass/',
-  IGNORE: [
-    '!**/.#*',
-    '!**/flycheck_*'
+  IGNORE: ['!**/.#*', '!**/flycheck_*'],
+  // @@@ remove these as they are migrated
+  IGNORE_SASS: [
+    '!sass/animate/**/*.scss',
+    '!sass/color/**/*.scss',
+    '!sass/core/**/*.scss',
+    '!sass/layout/**/*.scss',
+    '!sass/scale/**/*.scss',
+    '!sass/tokens/**/*.scss',
+    '!sass/type/**/*.scss',
```

Comment:
@mirisuzanne Because of the way we're calling Sassdoc (via `gulpfile`), any src/exclude options needed to be defined here. I expect we may clean this up and/or remove gulp entirely before the breaking release anyway, but we can see. This should unblock you.

The remaining errors were coming from `tokens/`, so I added that to this list to get a clean build.